### PR TITLE
feat(cli-tools): [NET-972] raw subscription and metadata support to `subscribe` command

### PR DESCRIPTION
**Pull request by harbu**
## Summary

Add raw subscription support to command `stream subscribe` with flag `--raw` or `-r`. Messages will be printed encrypted to stdout.

Add metadata support to command `streamr subscribe` with flag `--with-metadata` or `-m`. Messages will be printed with their associated metadata (publisherId, timestamp, sequenceNo etc.). 

## Checklist before requesting a review

- [x] Is this a breaking change? If it is, be clear in summary.
- [x] Read through code myself one more time.
- [x] Make sure any and all `TODO` comments left behind are meant to be left in.
- [x] Has reasonable passing test coverage?
- [x] Updated changelog if applicable.
- [ ] Updated documentation if applicable.

### CODE REVIEW

File: packages/cli-tools/bin/streamr-stream-subscribe.ts [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
 createClientCommand(async (client: StreamrClient, streamId: string, options: Options) => {
+    const formMessage = options.withMetadata
+        ? (message: unknown, metadata: MessageMetadata) => ({ message, metadata: omit(metadata, 'streamMessage') })
+        : (message: unknown) => message
     await client.subscribe({
         streamId,
         partition: options.partition,
-    }, (message) => console.info(JSON.stringify(message)))
+        raw: options.raw
+    }, (message, metadata) => console.info(JSON.stringify(formMessage(message, metadata))))
```
Comment (teogeb) [inline packages/cli-tools/bin/streamr-stream-subscribe.ts]: Do not stringify the hex output of encrypted message when using raw?

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,9 @@ found [here](packages/broker/CHANGELOG.md).
 
 #### Added
 
+- Add flag `--raw` (or shorthand `-r`) to command `stream subscribe` for raw subscription (https://github.com/streamr-dev/network/pull/1317)
```
Comment (teogeb) [inline CHANGELOG.md:30]: Mayve changelog items could be as short as possible? I.e. not mentioning the shorthands?

File: packages/cli-tools/test/stream-publish-subscribe.test.ts [hunk 3]
```diff
@@ -6,30 +6,79 @@ import { createTestClient, runCommand, startCommand } from './utils'
...
+    it('raw subscription', async () => {
+        const subscriberAbortController = new AbortController()
+        const subscriberOutputIterable = startCommand(`stream subscribe ${streamId} --raw`, {
+            privateKey: subscriberPrivateKey,
+            abortSignal: subscriberAbortController.signal,
+        })
+        publishViaCliCommand()
+        const receivedMessage = (await collect(subscriberOutputIterable, 1))[0]
+        subscriberAbortController.abort()
+        expect(receivedMessage).toMatch(/[0-9a-fA-F]+/)
```
Comment (teogeb) [inline packages/cli-tools/test/stream-publish-subscribe.test.ts]: Check that the whole input matches, not just a substring:

```suggestion
        expect(receivedMessage).toMatch(/^[0-9a-fA-F]+$/)
```

File: packages/cli-tools/bin/streamr-stream-subscribe.ts (hunk 1 above)
Comment (harbu) [reply packages/cli-tools/bin/streamr-stream-subscribe.ts]: fixed

**Review by teogeb — APPROVED** [review]

**Pull request merged**